webviz_config/_build_webviz.py

```diff
@@ -64,7 +66,7 @@ def build_webviz(args: argparse.Namespace) -> None:
                     "error message and traceback above"
                 )
 
-            os.remove(os.path.join(build_directory, "copy_data.py"))
+            os.remove(build_directory / "copy_data.py")
```

Comment:
Here we could perhaps use [`unlink`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.unlink).

```suggestion
            (build_directory / "copy_data.py").unlink()
```